README.md

```diff
@@ -105,28 +105,37 @@ pip install -e .
 
 ### Run an experiment
 
-There are two ways to run an experiment.
+There are two ways to run an experiment of multi-partner learning approaches and/or contributivity methods on a given dataset.
 
 #### (Recommended) The notebook approach
```

Comment:
I would remove the "notebook" because you can do it with a script. And you can use a notebook with the config file
So I would propose something like: Defining the experiment in the code

Reply:
Done